client/types.ts

```diff
@@ -64,6 +90,10 @@ interface ThemeFeature {
 	term_id: string;
 }
 
+export interface TrendingThemesFilter {
+	filter: string | undefined;
```

Comment:
Yes it will be if no filter is selected before the user selects the trending theme tab. 

Now that you bring it up though, I'm not sure if it's still even being used for anything meaningful. The filter appears to be disregarded in the reducer.



@blackjackkent could you confirm that we still want the `trendingThemesFilter`?

Reply:
@jeyip I believe you are correct - I think this is a copy/paste error that slipped through the cracks when following the pattern used by the `recommended-themes` actions. :)